enhancements/scheduling/prevent-user-workloads-from-being-scheduled-on-control-plane-nodes.md

```diff
@@ -229,60 +249,133 @@ spec:
+      operator: "Exists"
+      effect: "NoExecute"
+    - key: "node.kubernetes.io/network-unavailable"
+      operator: "Exists"
+      effect: "NoExecute"
+    - key: "node.kubernetes.io/kubelet-unreachable"
+      operator: "Exists"
+      effect: "NoExecute"
 ```
 
+- This pod needs to use the Service Account created before.
+
 - These requests are then submitted to the Kubernetes API server for processing.
 
-6. Admission Policy Enforcement:
+7. Admission Policy Enforcement:
 
 - Upon receiving pod deployment or update requests, the Kubernetes API server invokes the ValidatingAdmissionPolicy. It evaluates the pod's tolerations against the policy's defined CEL expressions, determining the request's compatibility with established configured guidelines.
 
-7. Policy Decision Feedback:
+8. Policy Decision Feedback:
 
 - Should a pod's request contradict the policy, it is promptly rejected. The User/Developer is informed of this decision through an error message that highlights the specific policy violation, akin to:
 
 ```
-The pods "my-pod" is invalid: : ValidatingAdmissionPolicy 'control-plane-scheduling-policy' with binding 'control-plane-scheduling-policy-binding' denied request: failed expression: object.spec.tolerations.all(toleration, !(toleration.key == 'node-role.kubernetes.io/control-plane' && toleration.effect == 'NoExecute'))
+The pods "my-pod" is invalid: : "Pod toleration for 'NoExecute' is not authorized for service account 'special-service-account' in namespace 'default'."
 ```
 
 This message clearly indicates the failed policy expression, aiding Users/Developers in understanding the reason behind the rejection and guiding them towards necessary adjustments.
 
-- Conversely, if the request aligns with the policy or falls within an exception due to namespace labeling, the process advances eventually adhering to Kubernetes' standard scheduling steps (or direct pod placement with spec.nodeName).
+- Conversely, if the request aligns with the policy or falls within an exception due RBAC permission grating, the process advances eventually adhering to Kubernetes' standard scheduling steps (or direct pod placement with spec.nodeName).
 
-8. Emergency Overrides and Adjustments:
+9. Emergency Overrides and Adjustments:
 
-- For emergency scenarios necessitating temporary deviations from the norm, Cluster Administrators have the option to swiftly modify namespace labels or adjust the ValidatingAdmissionPolicy and ValidatingAdmissionPolicyBinding resources. This flexibility ensures that critical operations can proceed unhindered, even under exceptional circumstances.
+- For emergency scenarios necessitating temporary deviations from the norm, Cluster Administrators have the option to swiftly allow new tolerations through the RBAC mechanism or adjust the ValidatingAdmissionPolicy and ValidatingAdmissionPolicyBinding resources. This flexibility ensures that critical operations can proceed unhindered, even under exceptional circumstances.
```

Comment:
The KSO might undo any change applied to the ValidatingAdmissionPolicy/ValidatingAdmissionPolicyBinding

Reply:
That's true, well change